app/models/service_template_transformation_plan_task.rb

```diff
@@ -245,6 +249,12 @@ def get_conversion_state
     update_options(updates)
   end
 
+  def cutover
+    unless conversion_host.create_cutover_file(options[:virtv2v_wrapper]['cutover_file'])
+      raise "Couldn't create cutover file for #{source.name} on #{conversion_host.name}"
```

Comment:
We should wrap this exception in gettext, e.g. `_("Couldn't...")` so that it can be localized